sdk/storage/storage-blob/test/node/appendblobclient.spec.ts

```diff
@@ -20,7 +20,7 @@ import { TokenCredential } from "@azure/core-http";
 import { assertClientUsesTokenCredential } from "../utils/assert";
 import { record } from "@azure/test-utils-recorder";
 import { Test_CPK_INFO } from "../utils/constants";
-dotenv.config({ path: "../.env" });
+dotenv.config();
```

Comment:
Are you sure about these?
This specific test file is inside the `node` folder, a layer below the `test` folder. Does that affect this?

Reply:
I believe it uses the .env available immediately from where tests are being executed. I still need to go through every project and confirm this.

Reply:
I will go one by one as soon as I can.

Reply:
I just checked, the .env file works well at the root of `storage-blob` with these changes for both `unit-test` and `integration-test` 👍

Reply:
Dotenv resolves `.env` files relative to the process working directory, see:

https://github.com/motdotla/dotenv/blob/7301ac9be0b2c766f865bbe24280bf82586d25aa/lib/main.js#L78

So I think what Daniel has done here is correct.

Reply:
Nice.